app/models/user.rb

```diff
@@ -51,6 +51,21 @@ def committee_member_pending_approval?
     professor? or judge? and !committee_member?
   end
 
+  def judge_or_professor_pending_approval?
+    if professor?
+      professor = self.professors.first
+      return (professor and not professor.approved)
+    elsif judge?
+      judge = self.judges.first
+      return (judge and not professor.approved)
```

Comment:
Acá me tira `undefined method 'approved' for nil:NilClass` cuando intento editar la cuenta de juez, se me hace que en la linea 60 debería ser `...and judge.approved` a lo mejor se te pasó cambiar esa man.

Reply:
Ya quedó, vaquero